eng/swagger_to_sdk_config.json

```diff
@@ -2,7 +2,15 @@
   "$schema": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/documentation/sdkautomation/SwaggerToSdkConfigSchema.json",
   "generateOptions": {
     "generateScript": {
-      "path": "pwsh ./eng/scripts/Invoke-GenerateAndBuildV2.ps1"
+      "path": "pwsh ./eng/scripts/Invoke-GenerateAndBuildV2.ps1",
+      "logPrefix": "DotNet-Sdk-Automation",
+      "stderr": {
+        "showInComment": ".*(\\[ERROR\\]|error NU|(?!0\\b)\\d+ Error\\(s\\)|ExternalError).*",
+        "scriptError": "\\[ERROR\\]"
```

Comment:
Do we need to also include errors that come from msbuild which generally look like "error: "?